pkg/kubelet/kubelet_pods.go

```diff
@@ -1831,6 +1838,57 @@ func (kl *Kubelet) convertToAPIContainerStatuses(pod *v1.Pod, podStatus *kubecon
 			oldStatusPtr = &oldStatus
 		}
 		status := convertContainerStatus(cStatus, oldStatusPtr)
+		if utilfeature.DefaultFeatureGate.Enabled(features.InPlacePodVerticalScaling) {
+			if status.State.Running != nil {
+				var requests, limits v1.ResourceList
+				// oldStatus should always exist if container is running
+				oldStatus, oldStatusFound := oldStatuses[cName]
+				// Set initial limits from container's spec upon transition to Running state
+				// For cpu & memory, limits queried from runtime via CRI always supercedes spec.limit
+				// For ephemeral-storage, a running container's status.limit equals spec.limit
+				determineResource := func(rName v1.ResourceName, ctrStsRes, ctrRes, oldStsRes, resource v1.ResourceList) {
```

Comment:
Use full variable name, it is important for people to understand which is which: `ctrStsRes, ctrRes, oldStsRes`

Reply:
fix is on the way in next commit.